supertokens_python/logger.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+#
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License") as published by the Apache Software Foundation.
+#
+# You may not use this file except in compliance with the License. You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# pyright: reportUnknownMemberType=false
+
+import json
+import logging
+from datetime import datetime
+from os import getenv
+from typing import TextIO, Union
+
+from .constants import VERSION
+
+# Configure logger
+logger = logging.getLogger()
+log_level_str = getenv('LOG_LEVEL', "").lower()
+logger.setLevel(logging.DEBUG if log_level_str == "debug" else logging.INFO)
+
```

Comment:
Add a comment showing how the logging output looks like (See other SDks)

Reply:
Done.